### PR TITLE
Use translatable string for maps error message

### PR DESCRIPTION
**Pull request by barryceelen**
Properly add the ‘Google Maps API not loaded’ error message (introduced in #318) to `CMBGmaps.strings` in stead of hard coding it in the JavaScript file. Additionally adds html escaping.

### CODE REVIEW

File: js/field-gmap.js [hunk 1]
```diff
@@ -3,15 +3,16 @@
...
-		$('.map').css('textAlign','center').append( 'Google Maps API not loaded.' );
-		return;
-	}
-
 	var CMBGmapsInit = function( fieldEl ) {
 
+		var mapCanvas = $('.map', fieldEl ).get(0);
+
+		if ( 'undefined' == typeof google ) {
+			$('<div>'+CMBGmaps.strings.googleMapsApiNotLoaded+'</div>').css({'width':'100%','textAlign':'center','marginTop':'1em'}).appendTo( mapCanvas );
```
Comment (mikeselander) [inline js/field-gmap.js]: Could you adjust the spacing here? Need spaces around the strings, concatenation `+`s, etc.

File: js/field-gmap.js [hunk 2]
```diff
@@ -3,15 +3,16 @@
...
+			return;
+		}
+
 		var searchInput = $('.map-search', fieldEl ).get(0);
-		var mapCanvas   = $('.map', fieldEl ).get(0);
```
Comment (mikeselander) [inline js/field-gmap.js]: Please add a space before the opening tick here. And, I know it's not your PR scope, but you could adjust the ones above and below I'd really appreciate it!

File: js/field-gmap.js [hunk 3]
```diff
@@ -3,15 +3,16 @@
...
 (function($) {
 
-	if ( 'undefined' == typeof google ) {
-		$('.map').css('textAlign','center').append( 'Google Maps API not loaded.' );
-		return;
-	}
-
 	var CMBGmapsInit = function( fieldEl ) {
 
+		var mapCanvas = $('.map', fieldEl ).get(0);
```
Comment (mikeselander) [inline js/field-gmap.js]: Please add a space before the opening tick here.

**Comment by mikeselander** [comment]
Hi Barry, thank you for the PR! I left a few notes n some very small spacing issues but otherwise, looks good! If you could adjust the spacing we'll get this in!

**Comment by barryceelen** [comment]
Hi Mike, spacing and some small other improvements added.

**Comment by mikeselander** [comment]
Barry, looks wonderful - thanks for the adjustments! Merging now.

**Pull request merged**